debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -681,6 +692,10 @@ public String getHeartbeatActionQuery() {
         return heartbeatActionQuery;
     }
 
+    public String getUnavailableValuePlaceholder() {
```

Comment:
Can't it also be a byte array (at least for Postgres)? Or is this handled over in the PG connector specifically?

Reply:
You're right, so I've changed the method on the `RelationalDatabaseConnectorConfig` to return `byte[]` to match the method signature in PG and properly overridden the method in PG with its special `hex:` impl.  This should now work universally across Oracle, PG, and any connector that needs this in the future.